core/src/main/java/tc/oc/pgm/api/map/Phase.java

```diff
@@ -13,6 +13,11 @@ public Component toComponent() {
   }
 
   public static Phase of(String query) {
-    return query.startsWith("dev") ? DEVELOPMENT : PRODUCTION;
+    for (Phase phase : Phase.values()) {
+      if (phase.toString().toLowerCase().startsWith(query.toLowerCase())) {
```

Comment:
```suggestion
    query = query.toLowerCase(Locale.ROOT);
    for (Phase phase : Phase.values()) {
      if (phase.toString().toLowerCase(Locale.ROOT).startsWith(query)) {
```

When using toLowerCase always include the locale, otherwise bad things can happen on special languages where rules you may think make sense no longer apply. Also, lowercasing the query can be moved out of the loop